VERSION

```diff
@@ -1 +1 @@
-0.1
+0.0.1
```

Comment:
Sure we want a three number versioning here?

Reply:
I think starting with "MAJOR.MINOR.MICRO" and semantic versioning is probably the most common versioning pattern for working across teams. But we can change it when we actually release and after having had some feedback from other collaborators.

Reply:
:+1: